development/docker-compose.dev.yml

```diff
@@ -3,7 +3,7 @@
 # any override will need to include these volumes to use them.
 # see:  https://github.com/docker/compose/issues/3729
 ---
-version: "3"
+version: "3.4"
```

Comment:
Why these changes?

Reply:
Didnt mean to commit those. I was getting docker errors until i changed the version

Reply:
Planning to revert this? Or is it something that needs to be changed?